src/Transport/Config/AzureServiceBusTransportInfrastructure.cs

```diff
@@ -27,7 +27,19 @@ public override EndpointInstance BindToLocalEndpoint(EndpointInstance instance)
 
         public override string ToTransportAddress(LogicalAddress logicalAddress)
         {
-            return logicalAddress.ToString();
+            var queue = new StringBuilder(logicalAddress.QueueName);
```

Comment:
Yes, in the current core code qualifier and discriminator are mutually exclusive. There is even a check in `LogicalAddress` which prevents creating an address with both. The idea is that the satellite queues (which have qualifier) are always shared by all instances.

I am not entirely sure if this is a permanent property but it seems unlikely that we'll have a per-instance satellites any time soon.